src/Data/List/Base.agda

```diff
@@ -369,6 +369,22 @@ initLast (x ∷ xs)         with initLast xs
 ... | []       = [] ∷ʳ' x
 ... | ys ∷ʳ' y = (x ∷ ys) ∷ʳ' y
 
+------------------------------------------------------------------------
+-- Splitting a list
+
+wordsBy : ∀ {P : Pred A p} → Decidable P → List A → List (List A)
```

Comment:
I was just about to merge this in when it occurred to me that this is usually called `split` rather than `wordsBy` in most languages right?